### PR TITLE
DOP-4359: add LanguagesBucket for s3

### PR DESCRIPTION
**Pull request by mmeigs**
### Stories/Links:

DOP-4359

### Notes

This adds the necessary S3 Buckets for `docs-languages`

It also removes the intgr buckets that don't even exist in S3...?

### README updates

- - [ ] This PR introduces changes that should be reflected in the README, and I have made those updates.
- - [x] This PR does not introduce changes that should be reflected in the README

### CODE REVIEW

File: infrastructure/ecs-main/serverless.yml [hunk 1]
```diff
@@ -121,11 +121,9 @@ custom:
   cSharpBucketName: docs-csharp-${self:provider.stage}
   cSharpIntgrBucketName: docs-csharp-dev
   goBucketName: docs-go-${self:provider.stage}
-  goIntgrBucketName: docs-go-intgr
   javaBucketName: docs-java-${self:provider.stage}
-  javaIntgrBucketName: docs-java-intgr
   nodeBucketName: docs-node-${self:provider.stage}
-  nodeIntgrBucketName: docs-node-intgr
```
Comment (rayangler) [inline infrastructure/ecs-main/serverless.yml:128]: Thanks for cleaning this up!

File: infrastructure/ecs-main/serverless.yml [hunk 2]
```diff
@@ -121,11 +121,9 @@ custom:
...
+  languagesBucketName: docs-languages-${self:provider.stage}
```
Comment (rayangler) [inline infrastructure/ecs-main/serverless.yml:126]: Do we need to add this bucket to the CDK's [list of buckets](https://github.com/mongodb/docs-worker-pool/blob/f75b3fc36a387b985ab5d5ad2bf633774edc70ef/cdk-infra/utils/buckets.ts#L7)?

**Review by rayangler — COMMENTED** [review]
Overall lgtm, but wanted to flag one question

File: infrastructure/ecs-main/serverless.yml (hunk 2 above)
Comment (mmeigs) [reply infrastructure/ecs-main/serverless.yml:126]: Great catch!! Thank you. Just added

**Review by rayangler — APPROVED** [review]

**Pull request merged**